include/battle.h

```diff
@@ -54,11 +54,15 @@ struct __attribute__((packed, aligned(2))) BattleMoveEffect
     const u8 *battleScript;
     u16 battleTvScore:3;
     u16 encourageEncore:1;
+    u16 twoTurnEffect:1;
     u16 semiInvulnerableEffect:1;
-    u16 flags:11; // coming soon...
+    u16 usesProtectCounter:1;
+    u16 forcePressure:1;
```

Comment:
I feel `forcePressure` should be set for moves instead of effects.

Reply:
Which means that it could be interpreted either way, and when that happens, I rather keep it for moves to allow more customization.